core/src/cluster_info_vote_listener.rs

```diff
@@ -409,17 +426,23 @@ impl ClusterInfoVoteListener {
             if time_since_lock.elapsed().as_millis() > GOSSIP_SLEEP_MILLIS as u128 {
                 let bank = poh_recorder.lock().unwrap().bank();
                 if let Some(bank) = bank {
-                    let last_version = bank.last_vote_sync.load(Ordering::Relaxed);
```

Comment:
looks like can remove `last_vote_sync` from `bank` too

Reply:
yup, good eye. I got lazy and didn't remove this. Removed here: https://github.com/solana-labs/solana/pull/20873/commits/1b433970d1e3076ce557cc0fda24f0b41027923d